### ISSUE TITLE
Improve Flux combined maintainers list

### ISSUE THREAD

**Issue by scottrigby** [fluxcd/community#135]
Follow-up on #134 

- [x] PR to update [CNCF project maintainers CSV](https://github.com/cncf/foundation/blob/master/project-maintainers.csv.)
- [ ] Change formatting of each repo's MAINTAINERS file to an automatically ingestable format
    - Ensure important info like company and contact info is still accessible somehow from this file. Either a linked, separate file or add to metadata somehow that still allows machines to read these
    - Ensure alumni are also recorded in MAINTAINERS files, but should not be confused in combined file with current maintainers
- [ ] Automate the combined `flux-project-maintainers.yaml` file from those machine-readable MAINTAINERS files
    - warn on differences or failure

**Comment by dholbach** [comment]
@scottrigby Unfortunately there's https://github.com/yaml/yaml/issues/48 so merging of sequences doesn't quite work like implemented in #134 - I tried writing a python script to generate some of the info, but it failed loading your YAML.

**Comment by dholbach** [comment]
To illustrate the problem:

```py
community on  main [?] 
❯ python -c "import yaml; yaml.safe_load(open('project/flux-project-maintainers.yaml').read())"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/daniel/.local/lib/python3.8/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/home/daniel/.local/lib/python3.8/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/home/daniel/.local/lib/python3.8/site-packages/yaml/constructor.py", line 51, in get_single_data
    return self.construct_document(node)
  File "/home/daniel/.local/lib/python3.8/site-packages/yaml/constructor.py", line 60, in construct_document
    for dummy in generator:
  File "/home/daniel/.local/lib/python3.8/site-packages/yaml/constructor.py", line 413, in construct_yaml_map
    value = self.construct_mapping(node)
  File "/home/daniel/.local/lib/python3.8/site-packages/yaml/constructor.py", line 217, in construct_mapping
    self.flatten_mapping(node)
  File "/home/daniel/.local/lib/python3.8/site-packages/yaml/constructor.py", line 194, in flatten_mapping
    raise ConstructorError("while constructing a mapping",
yaml.constructor.ConstructorError: while constructing a mapping
  in "<unicode string>", line 42, column 3:
    - <<: *flux2
      ^
expected a mapping for merging, but found scalar
  in "<unicode string>", line 13, column 3:
    - relu
      ^

community on  main [?] 
❯ 

```
